lib/jsdom/living/window/Screen-impl.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+// https://drafts.csswg.org/cssom-view-1/#the-screen-interface
```

Comment:
I think we usually have such URL comments in the IDL file. This is totally fine by me, but just wanted to know if that policy changed..

Reply:
I believe they're mainly used for partials within IDL files. That said, I would not mind making it standard practice to have them in both IDL and the implementation. They're often handy for looking up implementation details in the current context.

Reply:
I think we usually do IDL source link in the IDL file, implementation detail links in impl.